### PR TITLE
email notification

### PR DESCRIPTION
**Pull request by anna-kuzii**

### CODE REVIEW

File: src/server/data/emails/registration.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+export const registrationMail = (userEmail) => {
+  const currentDate = new Date().getDate();
+  const currentMonth = getMonth(new Date().getMonth());
+  const currentYear = new Date().getFullYear();
+
+  return {
+    from: process.env.EMAIL,
+    to: userEmail,
+    subject: 'Success registration',
+    html: `<table style="border-collapse: collapse; background: #f9f9fb; position: absolute; width: 100%; right: 50%; transform: translateX(50%); border-bottom: 50px solid #f9f9fb;">
```
Comment (InnaKutsik) [inline src/server/data/emails/registration.js:10]: move to const ---> html templates

File: src/server/helpers/sendEmail.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const nodemailer = require('nodemailer');
+
+export default async function sendEmail(emailTempate) {
+  let transporter = nodemailer.createTransport({
```
Comment (InnaKutsik) [inline src/server/helpers/sendEmail.js]: let ---> const

File: src/server/data/emails/registration.js (hunk 1 above)
Comment (anna-kuzii) [reply src/server/data/emails/registration.js:10]: rewrite

Comment (anna-kuzii) [reply src/server/data/emails/registration.js]: rewrite

File: src/server/helpers/sendEmail.js (hunk 2 above)
Comment (anna-kuzii) [reply src/server/helpers/sendEmail.js]: changed

**Review by InnaKutsik — APPROVED** [review]

**Review by khrystynapylypets — APPROVED** [review]

**Pull request merged**